### ISSUE TITLE
Implement advanced segmentation loss/metrics

### ISSUE THREAD

**Issue by maju116** [maju116/pyplatypus#15]
Based on https://www.kaggle.com/code/bigironsphere/loss-function-library-keras-pytorch/notebook implement:
1. Focal Loss (With CCE instead of BCE)
2.  Tversky coef
3. Tversky Loss (1 - Tversky coef)
4. Focal Tversky Loss
5. Lovasz Hinge Loss
6. Combo Loss

Add methods, tests and if needed additional arguments `segmentation_loss.__init__`
